postgres-appliance/scripts/configure_spilo.py

```diff
@@ -627,11 +625,14 @@ def write_crontab(placeholders, overwrite):
                 return logging.warning('Cron is already configured. (Use option --force to overwrite cron)')
 
     lines = ['PATH={PATH}'.format(**placeholders)]
-    lines += ['{BACKUP_SCHEDULE} /scripts/postgres_backup.sh "{WALE_ENV_DIR}" "{PGDATA}" "{BACKUP_NUM_TO_RETAIN}"'
-              .format(**placeholders)]
+
+    if bool(placeholders.get('USE_WALE')):
```

Comment:
one of the changes to crontab seems to lead to:

> 2018-09-06 11:51:06,274 - bootstrapping - INFO - Configuring crontab                                                   
"-":1: bad minute                                                                                         
errors in crontab file, can't install.

for example when the image was run as:
```bash
./build.sh --build-arg COMPRESS=true --tag spilo-test-walg .
docker run -it -e LOG_S3_BUCKET="test-bucket"  spilo-test-walg:latest
```
with 
```bash
docker run -it -e WAL_S3_BUCKET="test-bucket"  spilo-test-walg:latest
```
this error is not logged, but the crontab still seems to be empty for the user `postgres`